src/index.html

```diff
@@ -1,15 +0,0 @@
-<!DOCTYPE html>
```

Comment:
This is a really important file - make sure it doesn't get deleted! This is where the main project page (probably your welcome.html) belongs. The rest you can keep as is and move right into the src folder.